generators/2.0/java/templates/spring/src/main/java/GifFormActivity.java.ejs

```diff
@@ -0,0 +1,49 @@
+package <%= basePackage %>;
+
+import com.symphony.bdk.core.activity.ActivityMatcher;
+import com.symphony.bdk.core.activity.command.CommandContext;
+import com.symphony.bdk.core.activity.form.FormReplyActivity;
+import com.symphony.bdk.core.activity.form.FormReplyContext;
+import com.symphony.bdk.core.activity.model.ActivityInfo;
+import com.symphony.bdk.core.activity.model.ActivityType;
+import com.symphony.bdk.core.service.MessageService;
+import com.symphony.bdk.spring.annotation.Slash;
+import com.symphony.bdk.template.api.TemplateException;
+import lombok.extern.slf4j.Slf4j;
```

Comment:
for generated code used by customers I would get rid of lombok because it might require them to do additional setups in their IDE (like enable annotation processing)
We could also get rid of commons lang as spring has a string utils class too I think